pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -120,6 +122,7 @@ const (
 	NodeProxierRoleBindingName        = NodeProxierRoleName + "s"
 	NodeAdminRoleBindingName          = NodeAdminRoleName + "s"
 	NodeReaderRoleBindingName         = NodeReaderRoleName + "s"
+	NodeConfigReaderRoleBindingName   = NodeConfigReaderRoleName + "s"
```

Comment:
No more `+ "s"` please.

Reply:
Why is it still there for the others?

Reply:
> Why is it still there for the others?

Legacy.  We stopped doing it upstream.

Reply:
Where's the comment in the code saying that?

Reply:
We should probably separate into section with header and footer to that effect